packages/webapp/src/routes/__tests__/land/management-monitoring-start-date.spec.js

```diff
@@ -161,7 +161,7 @@ describe(url, () => {
 
           await postHandler({ payload, yar: session }, h)
           expect(viewArgs).toEqual('')
-          expect(redirectArgs[0]).toEqual('/land/check-and-submit')
+          expect(redirectArgs[0]).toEqual('land/management-monitoring-start-date')
```

Comment:
This is wrong. I think you updated this expect line to try to fix the test that was broken in master? This change needs undoing and the branch rebasing with master to pull in the fix for the test, which was to set a date in the session. The test should redirect to `/land/check-and-submit` if it is working correctly